README.md

```diff
@@ -123,23 +123,11 @@ If a runner appears offline in the [runners settings page](https://github.com/ho
 
 Update the `flake.lock` based on your changes
 ```shell
-nix flake lock --update-input nixpkgsGithubActionRunners
+nix flake update --refresh nixpkgsGithubActionRunners
 ```
 
-If that works, check that everything builds successfully on the builder
```

Comment:
This doesn't work... Spent quite a while working out why. Looking at the build logs that were failing to change the state of the remote runner, I see

```
+ ssh root@95.217.193.35 nixos-rebuild -j4 switch --refresh --flake github:holochain/holochain-infra/deploy/linux-builder-01#linux-builder-01
```

So it always deploys from a branch. I do now remember that change being made but the docs didn't get updated...